### PR TITLE
websocket: use namespace in broadcast message as well

### PR DESCRIPTION
**Pull request by safchain**

### CODE REVIEW

**Comment by safchain** [comment]
run skydive-cdd-overview-tests
run skydive-functional-tests-backend-elasticsearch

**Comment by safchain** [comment]
run skydive-functional-tests-backend-elasticsearch

**Comment by safchain** [comment]
run skydive-functional-tests-backend-elasticsearch

**Comment by safchain** [comment]
run skydive-functional-tests-backend-elasticsearch

File: flow/server/flow_subscriber_endpoint.go [hunk 1]
```diff
@@ -18,25 +18,72 @@
...
+	}
+}
+
+// OnConnected Server interface
+func (fs *FlowSubscriberEndpoint) OnConnected(c ws.Speaker) {
+	ns := flowNS
+
+	if namespaces, ok := c.GetHeaders()["X-Websocket-Namespace"]; ok {
+		if len(namespaces) > 0 {
+			ns = namespaces[0]
```
Comment (masco) [inline flow/server/flow_subscriber_endpoint.go]: How about adding all the namespaces here with the same speaker if the subscriber gives more than one NS
``` 
for _, namespace := range namespaces {
fs.nsSubscriber[namespace] = c
}
```
Just a doubt, not sure whether it is useful or not :-)

File: flow/server/flow_subscriber_endpoint.go [hunk 2]
```diff
@@ -18,25 +18,72 @@
...
+}
+
+// OnDisconnected Server interface
+func (fs *FlowSubscriberEndpoint) OnDisconnected(c ws.Speaker) {
+	fs.Lock()
+	defer fs.Unlock()
+
+	for k, nc := range fs.nsSubscriber {
+		if nc == c {
+			delete(fs.nsSubscriber, k)
```
Comment (masco) [inline flow/server/flow_subscriber_endpoint.go]: if only one namespace per subscriber, the above comment is not valid so we can return here as soon as we delete the entry.

File: flow/server/flow_subscriber_endpoint.go (hunk 1 above)
Comment (safchain) [reply flow/server/flow_subscriber_endpoint.go]: Indeed, going to adapt

**Comment by safchain** [comment]
run skydive-cdd-overview-tests
run skydive-functional-tests-backend-elasticsearch

**Comment by safchain** [comment]
run skydive-cdd-overview-tests

**Pull request merged**